GitUI/CommandsDialogs/WorktreeDialog/FormManageWorktree.cs

```diff
@@ -246,7 +246,7 @@ private bool IsCurrentlyOpenedWorktree(WorkTree workTree)
 
         private void buttonCreateNewWorktree_Click(object sender, EventArgs e)
         {
-            using FormCreateWorktree formCreateWorktree = new(UICommands);
+            using FormCreateWorktree formCreateWorktree = new(UICommands, _worktrees[0].Path);
```

Comment:
Is there guarantee the array has items?

Reply:
The original clone must exist

Reply:
Like @gerhardol said...there is always at least the line of the cloned repository